manifest_staging/charts/aad-pod-identity/templates/role.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "aad-pod-identity.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "aad-pod-identity.labels" . | nindent 4 }}
+rules:
+  - apiGroups: ['extensions']
+    resources: ['podsecuritypolicies']
+    verbs:     ['use']
+    resourceNames:
+      - aad-pod-identity
```

Comment:
This should point to the name of the `PodSecurityPolicy`

Reply:
Fixed